frontend/src/metabase/actions/components/ActionCreator/FormCreator/FormCreator.tsx

```diff
@@ -56,103 +40,29 @@ export function FormCreator({
     }
   }, [params, formSettings]);
 
-  const handleChangeFieldSettings = (
-    paramId: string,
-    newFieldSettings: FieldSettings,
-  ) => {
-    setFormSettings({
-      ...formSettings,
-      fields: {
-        ...formSettings.fields,
-        [paramId]: newFieldSettings,
-      },
-    });
-  };
-
-  const handleDragEnd: OnDragEndResponder = ({ source, destination }) => {
-    const oldOrder = source.index;
-    const newOrder = destination?.index ?? source.index;
-
-    const reorderedFields = reorderFields(
-      formSettings.fields,
-      oldOrder,
-      newOrder,
-    );
-    setFormSettings({
-      ...formSettings,
-      fields: reorderedFields,
-    });
-  };
-
   const sortedParams = useMemo(
     () => params.sort(sortActionParams(formSettings)),
     [params, formSettings],
   );
 
+  if (!sortedParams.length) {
+    return (
+      <FormCreatorWrapper>
+        <EmptyFormPlaceholder onExampleClick={onExampleClick} />
+      </FormCreatorWrapper>
+    );
+  }
+
   return (
     <FormCreatorWrapper>
-      <DragDropContext onDragEnd={handleDragEnd}>
-        <Droppable droppableId="action-form-droppable">
-          {(provided: DroppableProvided) => (
-            <div {...provided.droppableProps} ref={provided.innerRef}>
-              {sortedParams.map((param, index) => (
-                <Draggable
-                  key={`draggable-${param.id}`}
-                  draggableId={param.id}
-                  index={index}
-                >
-                  {(provided: DraggableProvided) => (
-                    <div
-                      ref={provided.innerRef}
-                      {...provided.draggableProps}
-                      {...provided.dragHandleProps}
-                      className="mb1"
-                    >
-                      <FormItem
-                        key={param.id}
-                        param={param}
-                        fieldSettings={
-                          formSettings.fields?.[param.id] ??
-                          getDefaultFieldSettings()
-                        }
-                        onChange={(newSettings: FieldSettings) =>
-                          handleChangeFieldSettings(param.id, newSettings)
-                        }
-                      />
-                    </div>
-                  )}
-                </Draggable>
-              ))}
-              {!params.length && (
-                <EmptyFormPlaceholder onExampleClick={onExampleClick} />
-              )}
-            </div>
-          )}
-        </Droppable>
-      </DragDropContext>
+      <ActionForm
+        parameters={sortedParams}
+        initialValues={[]}
+        onClose={_.noop}
+        onSubmit={_.noop}
```

Comment:
Why do we pass `noop` here?

Reply:
Until we tackle https://github.com/metabase/metabase/issues/26701, we don't even show the submit/cancel buttons in the form creator.

It's a little easier type-wise to pass `_.noop` than to make these functions optional.